src/geo/transform.js

```diff
@@ -295,6 +297,9 @@ class Transform {
      * @returns {number} pixel coordinate
      */
     latY(lat: number) {
+        if (lat > this.maxValidLatitude || lat < this.maxValidLatitude * -1) {
```

Comment:
Is there any point in using an `if` here? Why not just call clamp directly?

Reply:
That's probably fine. Originally, I had a console warning inside the if statement but I think we've decided to favor documentation over a warning so this can be removed.